torch/csrc/autograd/python_variable.cpp

```diff
@@ -525,9 +527,9 @@ static PyObject* THPVariable_as_subclass(PyObject* _self, PyObject* args, PyObje
 static PyObject* THPVariable_make_subclass(PyObject* _ignored, PyObject* args, PyObject* kwargs) {
   HANDLE_TH_ERRORS
   static PythonArgParser parser({
-    "_make_subclass(PyObject* cls, Tensor data, bool require_grad=False)",
+    "_make_subclass(PyObject* cls, Tensor data, bool require_grad=False, bool use_custom_strides=False)",
```

Comment:
This should be a kwarg only; also minor bikeshed I suggest the name `dispatch_strides` instead.